packages/components/src/components/domainproperties/dataset/DatasetPropertiesPanelFormElements.tsx

```diff
@@ -0,0 +1,297 @@
+/*
+ * Copyright (c) 2020 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import {Col, Radio, Row} from 'react-bootstrap';
+import {DatasetModel} from './models';
+import {LabelHelpTip, SelectInput} from "../../..";
+import {Creatable} from 'react-select'
+import {fetchCategories} from "./actions";
+import {DatasetSettingsInput} from "./DatasetPropertiesAdvancedSettings";
+import {CheckBox} from "../list/ListPropertiesPanelFormElements";
+import "../../../theme/dataset.scss";
+
+interface BasicPropertiesInputsProps {
+    model?: DatasetModel;
+    onInputChange: (any) => void;
+    name?: string;
+    description?: string;
+    categoryId?: number;
+    onCategoryChange?: any;
+    label?: string;
+}
+
+interface BasicPropertiesTitleProps {
+    title: string
+}
+
+export class BasicPropertiesTitle extends React.PureComponent<BasicPropertiesTitleProps> {
+    render() {
+        return <div className="domain-field-section-heading">{this.props.title}</div>;
+    }
+}
+
+export class DescriptionInput extends React.PureComponent<BasicPropertiesInputsProps> {
```

Comment:
If this component is an exact copy of the list version, can we do a little refactoring so they can both use the same component?